### PR TITLE
Added Setup.py for pypi release 

### PR DESCRIPTION
**Pull request by ashish493**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as readme_file:
+    readme = readme_file.read()
+
+requirements = ["casbin==0.8.4", "psycopg2-binary==2.8.6", "black==20.8b1"]
+
+setup(
+    name="postgresql-watcher",
+    version="0.0.1",
```
Comment (ashish493) [inline setup.py:10]: Once @Zxilly completes the CI in the other PR,  we can change the version through the CI for every release.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+with open("README.md", "r") as readme_file:
+    readme = readme_file.read()
+
+requirements = ["casbin==0.8.4", "psycopg2-binary==2.8.6", "black==20.8b1"]
+
+setup(
+    name="postgresql-watcher",
+    version="0.0.1",
+    author="hsluoyz",
+    author_email="hsluoyz@gmail.com",
```
Comment (ashish493) [inline setup.py:12]: @hsluoyz could u publish it on PyPi through your credentials?

Comment (Zxilly) [reply setup.py:12]: In fact, currently this package not released.
You can release it, but plz add casbin to maintainer after that.

Comment (Zxilly) [reply setup.py:12]: Release it now is not a good idea. Since this package didn't complete yet. I will later working on this

Comment (ashish493) [reply setup.py:12]: yeah, no problem. I will look into other issues.

**Pull request merged**